app/controllers/admin_request_controller.rb

```diff
@@ -5,10 +5,14 @@
 # Email: hello@mysociety.org; WWW: http://www.mysociety.org/
 
 class AdminRequestController < AdminController
+  include Admin::Sortable
+
   before_action :set_info_request, :check_info_request, only: %i[
     show edit update destroy move generate_upload_url hide
   ]
 
+  sortable default: :updated_at_desc, only: [:index]
```

Comment:
Did you mean to change the sort order from `created_at desc` here?

Reply:
The current sort order is updated at desc, so yeah kept it as that for now